committers.md

```diff
@@ -86,3 +86,44 @@ non-voting project maintainers.
     {% endif %}
   {% endfor %}
 </tbody></table>
+
+### **Becoming a committer**
```

Comment:
Tiny comment: is the bold font needed here? (the ``**``) The other titles above don't have this

Reply:
That is an excellent point 👍 

```suggestion
### Becoming a committer
```